gatsby/copyingplonesite.rst

```diff
@@ -0,0 +1,171 @@
+Copying The Plone Site
+======================
+
+The end goal of the plugin along with GatsbyJS is to generate a static site which is an exact copy of the Plone site it sourced data from.
+
+Breaking this into steps:
+- Create pages for each content object and ensure tree structure by retaining parent and items relationships
+- Display HTML, images and files in content objects correctly
+- Handle navigation in the site with a Navbar and Breadcrumbs Breadcrumbs
+
+
+Page Creation
+-------------
+
+``exports.createPages`` is the API used in ``gatsby-node.js`` for creating pages from nodes.
+
+For instance, to create pages for all nodes of type ``PloneFolder``:
```

Comment:
i think that should be cleaner to users, to alway say which file we are showing/editing